packages/runtime/container-runtime/src/opTelemetry.ts

```diff
@@ -49,7 +49,7 @@ export class OpTracker {
         });
 
         deltaManager.on("op", (message: ISequencedDocumentMessage) => {
-            this._nonSystemOpCount += isSystemMessage(message) ? 0 : 1;
+            this._nonSystemOpCount += !isRuntimeMessage(message) ? 0 : 1;
```

Comment:
`this._nonSystemOpCount += isRuntimeMessage(message) ? 1 : 0;`
Might be cleaner to remove the `!` and simplify the expression a little bit.